packages/roosterjs-editor-api/lib/cursor/getNodeAtCursor.ts

```diff
@@ -1,12 +1,16 @@
 import { Editor } from 'roosterjs-editor-core';
 import { NodeType } from 'roosterjs-editor-types';
 
+/**
+ * Get the node at selection
+ * if editor has focus, use selection.focusNode
+ * if for some reason, the focus node does not get us a good node
+ * fallback to this.getSelectionRange() which will return you a cached selection range if there is any
```

Comment:
Do use function name like getSelectionRange() here in comment since it doesn't help. Just try to describe what will be returned.

Reply:
Changed.